### ISSUE TITLE
Feature Request (for Custom size): Option to swap _width-height_ and _x-y_ on **vertical display**

### ISSUE THREAD

**Issue by marcelliino** [rxhanson/RectanglePro-Community#27]
Custom shortcuts should have the same behavior as the **defaults** on vertical display. (or at least give the option)
For example:
- sixths are divided into two rows on the minimum/shortest display length `min(WIDTH, HEIGHT)`.
- fourths are divided into four columns on the maximum display length `max(WIDTH, HEIGHT)` so they go sideways when horizontal and down when vertical.
- etc.

I think the simplest way to do this without any calculation is just to retain the display's width and height as horizontal orientation.
So the HEIGHT will always be shorter than the WIDTH even on vertical orientation. (tho I'm not sure this is possible).

**Comment by rxhanson** [comment]
My plan for handling screen orientations is to add a second set of inputs that would be applicable for vertical orientation if configured, since that would solve the generic case up front where users have proven to have different opinions about the size/pos of windows in each orientation. After the generic case is added in, I might consider adding a simpler way, like you mention with retaining display width and height ratios.

**Comment by marcelliino** [comment]
Also, it seems that Eighths doesn't work the same as Sixths on vertical orientation.

**Comment by rxhanson** [comment]
Yes, this is an artifact from Rectangle, where I figured it was worth it just to get Eighths in and fix the behavior when I get a chance.
